### PR TITLE
Empire Tiny Ship (GCI Task)

### PR DESCRIPTION
**Pull request by PAndaContron**
This PR adds a new Empire faction with a tiny ship and a start config

### CODE REVIEW

**Comment by GooeyHub** [comment]
Can one of the admins please verify this patch?

File: modules/empire/assets/configs/playerSpawnConfig.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "Empire Tiny": {
+	    "hull": "empire:empireTiny",
```
Comment (vampcat) [inline modules/empire/assets/configs/playerSpawnConfig.json]: These lines use a combination of tabs and spaces. We don't like tabs around these parts.

Please make sure that none of your files contain tabs.

**Comment by PAndaContron** [comment]
I drew the sprite for the ship myself.

**Comment by vampcat** [comment]
@GooeyHu: ok to test

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Pull request closed without merging**